JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -52,7 +52,7 @@ fun PostCardTop(post: Post) {
         post.image?.let { image ->
             Container(modifier = LayoutHeight.Min(180.dp) + LayoutWidth.Fill) {
                 Clip(shape = RoundedCornerShape(4.dp)) {
-                    DrawImage(image)
+                    SimpleImage(image = image)
```

Comment:
Ditto comment.

Reply:
Same note, had to add some TODO/remove code to get Clip working in dev06